docs/release.rst

```diff
@@ -11,9 +11,11 @@ Release notes
 Unreleased
 ----------
 
-Maintenance
```

Comment:
I think we need new blank sections for Enhancements, Maintenance, etc. under unreleased.

Reply:
Got it.